src/main/java/com/opentable/server/EmbeddedJettyBase.java

```diff
@@ -110,6 +110,13 @@
     @Value("${ot.httpserver.ssl-allowed-deprecated-ciphers:}")
     List<String> allowedDeprecatedCiphers;
 
+    // the following two values (shouldSleepBeforeShutdown, sleepDurationBeforeShutdown) help an application control whether they want a pause before jetty shutdown to ensure that the discovery unannounce has propagated to all clients requesting the application
+    @Value("${ot.httpserver.sleep-before-shutdown:false}")
```

Comment:
I think the default should be `true`.

Reply:
since the otj-server did not have a support for sleeping before Jetty shutdown as part of the original implementation, I kept it as `false`